tests/integration-tests/tests/scaling/test_scaling.py

```diff
@@ -59,48 +59,27 @@ def test_multiple_jobs_submission(scheduler, region, pcluster_config_reader, clu
     _assert_test_jobs_completed(remote_command_executor, max_jobs_execution_time * 60)
 
     logging.info("Verifying auto-scaling worked correctly")
-    _assert_scaling_works(
-        asg_capacity_time_series=ec2_capacity_time_series,
-        compute_nodes_time_series=compute_nodes_time_series,
-        expected_asg_capacity=(0, 3),
-        expected_compute_nodes=(0, 3),
-    )
+    _assert_scaling_works(compute_nodes_time_series=compute_nodes_time_series, expected_compute_nodes=(0, 3))
 
     logging.info("Verifying no error in logs")
     assert_no_errors_in_logs(remote_command_executor, scheduler)
 
 
-def _assert_scaling_works(
```

Comment:
I think we should not remove the asg part but just rename it cause it was already checking against ec2 capacity in case of Slurm and we should preserve that assertion

Reply:
Done, thanks!